graphql/resolvers.go

```diff
@@ -299,6 +299,97 @@ func (r *taskResolver) ReliesOn(ctx context.Context, at *restModel.APITask) ([]*
 	return dependencies, nil
 }
 
+func (r *taskResolver) DependsOn(ctx context.Context, at *restModel.APITask) ([]*Dependency, error) {
```

Comment:
What is this being used for?

Reply:
This will be replacing the `reliesOn` field. For some reason when we implemented a dependencies resolver the `dependsOn` field was renamed to `reliesOn`. This is resolving that and renaming it back to `dependsOn`